frontend/src/DialectReportView.tsx

```diff
@@ -1,15 +1,19 @@
 // @ts-ignore
-import CasesSection from "./components/Cases/CasesSection";
+import CasesSection from "./components/Cases/CasesSection.jsx";
```

Comment:
What's happening here?

Reply:
I don't actually know how the extension got added. 😅
Removed